src/main/src/main/java/org/geoserver/catalog/AttributeTypeInfo.java

```diff
@@ -99,4 +101,18 @@ public interface AttributeTypeInfo extends Serializable {
      * @param source A valid CQL expression, in the simple case, the attribute name
      */
     void setSource(String source);
+
+    /**
+     * Get Attribute description
+     *
+     * @return Attribute description
+     */
+    GrowableInternationalString getDescription();
```

Comment:
```suggestion
    InternationalString getDescription();
```

We just need the implementation to use GrowableInternationalString, XStream reflects directly on the fields of the Impl class, does not work against bean properties.